index.js

```diff
@@ -64,27 +67,34 @@ function checkGuess() {
 
 function hideAllMessages() {
   for (let elementIndex = 0; elementIndex <= messages.length; elementIndex++) {
-    messages[elementIndex].style.display = 'none';
+    messages[elementIndex] = 'none';
```

Comment:
We actually want to leave this line  with setting the style to 'none' alone, the bug in this function comes from setting up the for loop, 
it's checking `element <= messages.length` when it should be just the less than sign `elementIndex < messages.length`